src/components/sidebars/SidebarFeatures.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from "react";
+import SearchFeatures from "../../containers/search/SearchFeatures";
+import { createFeatureList } from "../../functions/createFeatureList";
+
+class SidebarFeatures extends Component {
+
+  render () {
+    return (
+      <div className="sidebar-features">
+        <SearchFeatures/>
+        {createFeatureList()}
```

Comment:
Shouldn't it be just using createFeatureList as element?

Reply:
Right, my bad. Done.